modules/nf-core/force/tileextent/main.nf

```diff
@@ -0,0 +1,43 @@
+process FORCE_TILEEXTENT {
+    tag "${aoi.simpleName}"
+    label 'process_single'
+
+    container "nf-core/force:3.8.01"
+
+    input:
+    path aoi
+    path "tmp/datacube-definition.prj"
+    path shapefile_dbf
```

Comment:
```suggestion
    path(shapefile_dbf, stageInMode: 'copy')
```
possibly this is what you want to do.

Reply:
Ok, so can't be set at the input level.
You could set it in the process level though, by setting `stageInMode 'copy'` underneath the tag, [several other modules do this](https://github.com/search?q=repo%3Anf-core%2Fmodules%20stageInMode&type=code).